Hercules-DJControl-Inpulse-500-script.js

```diff
@@ -1330,6 +1365,9 @@ DJCi500.updateDeckStatus = function(group) {
   DJCi500.fxSelIndicator(0, "[EffectRack1_EffectUnit1]", 0, 0);
   DJCi500.fxSelIndicator(0, "[EffectRack1_EffectUnit2]", 0, 0);
 
+  DJCi500.fxEnabledIndicator(0, "[QuickEffectRack1_[Channel3]]", 0, 0);
+  DJCi500.fxEnabledIndicator(0, "[QuickEffectRack1_[Channel4]]", 0, 0);
+

```

Comment:
Shouldn't these go with the Channel 1 and Channel 2 enabling calls in line 1130?

Reply:
From what I understand here is to update current group state. Ive pushed commit with fix here. From my manual testing I think it works fine now.